common/user/evaluate_roles.ts

```diff
@@ -23,6 +24,10 @@ export async function evaluateUserRoles(user: User): Promise<Role[]> {
         evaluateScreenerRoles(screener, roles);
     }
 
+    if (await isSSOUser(user.userID)) {
```

Comment:
Do we refresh the user session on loginPassword + register after the SSO secret was added?

Reply:
No, we do it before - so in createIdpLogin you would habe to refresh the session / manually add the SSO_USER role there